utensor_cgen/third_party/flatbuffers/builder.py

```diff
@@ -0,0 +1,764 @@
+# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
WTF @neil-tan, do you copy the entire flatbuffers python library in this PR!?
Why do you do this?

Reply:
The current version of Pypl flatbuffers package is missing the ability to add an identifier string to the model file. So, a copy of the latest flatbuffer (github) python api is included. The generated tflm Flatbuffer api, however, reference the standard import.

It would be great if we can make the latest flatbuffer importable, but I had to settle for Tods workaround now.

Reply:
I can't use github repo as dependency since the repo is not setuptools compatible.
Fuck that.

Though I can write an install hook which will clone and install latest version from the repo, that will make the install process slow.
My suggestion is copy the core library only and remove all auto generated files since that could be generated during installation.

Fuck, I'm so pissed.

Reply:
@dboyliao good proposal generating the TFLM binding as a part of the setup process btw, as discussed offline yesterday.

Reply:
Yay, I'll help you with that after merging #94.
BTW, I add legalization pass in the TF frontend.
It's part of the frontend but this is temporary.
I will refactor the legalization and lowering as separate phases for codegen in the future.

Reply:
Splendid